mm2src/mm2_main/src/rpc/dispatcher/dispatcher.rs

```diff
@@ -123,6 +123,7 @@ async fn dispatcher_v2(request: MmRpcRequest, ctx: MmArc) -> DispatcherResult<Re
         "add_delegation" => handle_mmrpc(ctx, request, add_delegation).await,
         "add_node_to_version_stat" => handle_mmrpc(ctx, request, add_node_to_version_stat).await,
         "best_orders" => handle_mmrpc(ctx, request, best_orders_rpc_v2).await,
+        "enable" => handle_mmrpc(ctx, request, enable_v2_wrapper).await,
```

Comment:
Hmm, I thought there will be lots of functions so I thought it would be more handy to handle them from single request just like enable v1. I got your point, will refactor it then.

Reply:
>  I thought it would be more handy to handle them from single request just like enable v1

Yeah, it actually worked out well when we had just UTXO and ETH protocols. But after some time, `lp_coins::lp_coininit` related to enable/electrum v1 started looking quite bloated. Imagine how it will look if we add like 10 more protocols :slightly_smiling_face:

Also, each coin protocol might have some specific request/response fields (and possible errors) not present for others (e.g. `swap_contract_address` for ETH/ERC20 that is irrelevant for UTXO). Mixing them up in a single struct might create problems, e.g.:
1. A field with similar meaning/same name might have different format for different protocols, which can require additional handling (maybe more complicated code).
2. If a change is required for one protocol, it may break other protocols unintentionally.

Considering these arguments, we decided that it would be better to have a separate method for each protocol. It's also easier to document coins activation this way (each RPC with a couple of examples, rather than single RPC with 10+ example of various protocols not fitting a single screen/page).

Reply:
> Considering these arguments, we decided that it would be better to have a separate method for each protocol. It's also easier to document coins activation this way (each RPC with a couple of examples, rather than single RPC with 10+ example of various protocols not fitting a single screen/page).

Yes, I definitely agree.